app/models/user.rb

```diff
@@ -10,4 +10,8 @@ class User < ApplicationRecord
   enum locale: { en: 0, es: 1 }
 
   include DeviseTokenAuth::Concerns::User
+
+  def display_name
```

Comment:
What is this method for?

Reply:
Active admin checks first for this method to display names of associations. Since I didn't have any of the other methods it looks for, I defined this one so my user associations would show the full name instead of this:

<img width="481" alt="screen shot 2018-10-01 at 3 45 31 pm" src="https://user-images.githubusercontent.com/12885969/46308843-12bb6180-c591-11e8-8b34-142386838f07.png">


https://github.com/activeadmin/activeadmin/blob/a252cebb4e6b4939775df2ef6920f16bec38f884/docs/3-index-pages/index-as-table.md#defining-columns

Reply:
Great. For this kind of methods we also use decorators with this gem: https://github.com/drapergem/draper